platform/ios/docs/guides/Info.plist Keys.md

```diff
@@ -18,7 +18,7 @@ The default value is `https://api.mapbox.com`.
 
 ## MGLMapboxMetricsEnabledSettingShownInApp
 
-If you have implemented custom opt-out of Mapbox Telemetry within the user interface of your app, use this key to disable the built-in check for opt-out support. See [this guide](https://www.mapbox.com/ios-sdk/#telemetry_opt_out) for more details.
+If you have implemented custom opt-out of Mapbox Telemetry within the user interface of your app, use this key to disable the built-in check for opt-out support. See [this guide](https://www.mapbox.com/help/how-mapbox-works/attribution/#mapbox-maps-sdk-for-ios) for more details.
```

Comment:
This page has moved [yet again](https://docs.mapbox.com/help/how-mapbox-works/attribution/#mapbox-maps-sdk-for-ios).